packages/opentelemetry-core/src/distribute-context/propagation/HttpTraceContext.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Carrier,
+  Context,
+  HttpTextFormat,
+  DistributedContext,
+} from '@opentelemetry/api';
+
+import {
+  getDistributedContext,
+  setDistributedContext,
+} from '../distribute-context';
+
+export const CORRELATION_CONTEXT_HEADER = 'Correlation-Context';
```

Comment:
Still very hesitant to use the W3C header while it is in such a state of change. I don't want to end up in a situation where we are deviated from the official spec, but still claiming to be using it. I would be much more comfortable with a vendor prefix of some sort, but that would require spec buy-in.

Reply:
What are our options here? Should we add a vendor prefix? is the best way to go?

Reply:
This spec PR which clears up this question I believe will merge today or tomorrow https://github.com/open-telemetry/opentelemetry-specification/pull/517

Reply:
Thanks to point me in the right direction!